Resources/Client/macOS/README.txt

```diff
@@ -0,0 +1,17 @@
+Hello! Thanks for downloading. I hope you enjoy.
+- Net_
+
+
+###################
+## Installation
+###################
+No installation is required, just run the application.
+
+
+###################
+## Credits
+###################
+Art by Penzilla - https://www.penzilladesign.com/
+Code by Net_

```

Comment:
Feel free to add a "macOS build originally by " here, if you'd like 😄 Up to you, you might want to avoid people looking you up if something breaks 😅